code/game/objects/items.dm

```diff
@@ -175,7 +175,6 @@ GLOBAL_DATUM_INIT(welding_sparks_prepdoor, /mutable_appearance, mutable_appearan
 		update_icon()
 
 /obj/item/Destroy()
-	flags_item &= ~NODROP //so the item is properly unequipped if on a mob.
```

Comment:
this was never required since `temporarilyRemoveItemFromInventory` gets called with `force=TRUE`